src/index.js

```diff
@@ -9,48 +9,76 @@ const verifyRepoConfig = async (repo, context) => {
   return false;
 };
 
-module.exports.workspaceActions = [{
-  label: 'Repo Sync - Export Workspace',
-  icon: 'fa-download',
-  action: async (context, models) => {
-    const repo = new WorkspaceRepo(context);
-    if (!await verifyRepoConfig(repo, context)) return;
-
-    const path = await repo.getPath();
-    const ex = await context.data.export.insomnia({
-      includePrivate: false,
-      format: 'yaml',
-      workspace: models.workspace,
-    });
-
-    fs.writeFileSync(`${path}/${models.workspace.name}.yml`, ex);
+module.exports.workspaceActions = [
+  {
+    label: 'Repo Sync - Export Workspace',
+    icon: 'fa-download',
+    action: async (context, models) => {
+      const repo = new WorkspaceRepo(context);
+      if (!(await verifyRepoConfig(repo, context))) return;
+
+      const path = await repo.getPath();
+      const rawJsonString = await context.data.export.insomnia({
+        includePrivate: false,
+        format: 'json',
+        workspace: models.workspace,
+      });
+
+      const exported = JSON.parse(rawJsonString);
+      
+      exported.resources.forEach((resource) => {
```

Comment:
maybe add a comment:
"open an issue if you see another relevant cleaning for insomnia export"